test/bigtest/tests/jobs-test.js

```diff
@@ -40,46 +40,50 @@ describe('Jobs lists', () => {
       expect(jobs.accordions(0).title.text).to.equal(translation.runningJobs);
     });
 
+    it('should display correct amount of running jobs', () => {
+      expect(runningJobs.jobItemsAmount).to.equal(3);
```

Comment:
Thanks for adding tests for multiple running jobs!

Reply:
Thank you.